COLLABORATOR_GUIDE.md

```diff
@@ -411,64 +411,45 @@ recommended but not required.
 
 ### Deprecations
 
-**Deprecation** refers to the identification of Public APIs that should no
-longer be used.
-
 Node.js uses three Deprecation levels:
 
-* *Documentation-Only Deprecation* refers to elements of the Public API that are
-  being staged for deprecation in a future Node.js major release. An explicit
-  notice indicating the deprecated status is added to the API documentation
-  but no functional changes are implemented in the code. There will be no
-  runtime deprecation warnings emitted for such deprecations by default.
-  Documentation-only deprecations may trigger a runtime warning when Node.js
-  is started with the [`--pending-deprecation`][] flag or the
-  `NODE_PENDING_DEPRECATION=1` environment variable is set.
-
-* *Runtime Deprecation* refers to the use of process warnings emitted at
-  runtime the first time that a deprecated API is used. A command-line
-  switch can be used to escalate such warnings into runtime errors that will
-  cause the Node.js process to exit. As with Documentation-Only Deprecation,
-  the documentation for the API must be updated to clearly indicate the
-  deprecated status.
-
-* *End-of-life* refers to APIs that have gone through Runtime Deprecation and
-  are no longer subject to the semantic versioning rules used by the project.
-  Backward-incompatible changes including complete removal of such APIs may
-  occur at any time.
-
-Documentation-Only Deprecations may be handled as semver-minor or semver-major
-changes. Such deprecations have no impact on the successful operation of running
-code and therefore should not be viewed as breaking changes.
```

Comment:
The semver part is important for us collaborators since it is otherwise not clear that a doc only deprecation may be `semver-minor`. I would be +1 for moving the notice up to the description of doc only deprecation.